client/executor/wasmtime/src/runtime.rs

```diff
@@ -251,6 +251,10 @@ directory = \"{cache_dir}\"
 fn common_config(semantics: &Semantics) -> std::result::Result<wasmtime::Config, WasmError> {
 	let mut config = wasmtime::Config::new();
 	config.cranelift_opt_level(wasmtime::OptLevel::SpeedAndSize);
+	#[cfg(feature = "jitdump")]
+	if std::env::var("WASMTIME_PROFILING_STRATEGY") == Ok("jitdump".to_owned()) {
```

Comment:
Should this env variable be added to https://github.com/paritytech/substrate/blob/master/utils/wasm-builder/src/lib.rs#L125 ?

Reply:
No. It should be documented as part of the crate docs.

Reply:
Sounds good, can you @librelois add documentation for the crate please ?

Reply:
Done